pkg/plugins/gittag/condition.go

```diff
@@ -0,0 +1,70 @@
+package gittag
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/scm"
+	"github.com/updatecli/updatecli/pkg/core/result"
+	"github.com/updatecli/updatecli/pkg/plugins/git/generic"
+)
+
+// Condition checks that a git tag exists
+func (gt *GitTag) Condition(source string) (bool, error) {
+	return gt.condition(source)
+}
+
+// ConditionFromSCM test if a tag exist from a git repository specific from SCM
+func (gt *GitTag) ConditionFromSCM(source string, scm scm.ScmHandler) (bool, error) {
+	path := scm.GetDirectory()
+
+	if len(gt.spec.Path) > 0 {
+		logrus.Warningf("Path is defined and set to %q but is overridden by the scm definition %q",
+			gt.spec.Path,
+			path)
+	}
+	gt.spec.Path = path
+
+	return gt.condition(source)
+}
+
+func (gt *GitTag) condition(source string) (bool, error) {
+	// If source input is empty, then it as disabled by the user with `disablesourceinput: true`
```

Comment:
There is a problem with "it as disabled"

Reply:
```suggestion
	// If source input is empty, then it means that it was disabled by the user with `disablesourceinput: true`
```